### DISCUSSION TITLE
WITHDRAWN: adding 200%,300% and 400% scaling options (@ cd78ebb)

### DISCUSSION THREAD

**Discussion by FulopNandor** [FulopNandor/dexed#11] [Announcements]
User @trancybabe opened the issue [#420](https://github.com/asb2m10/dexed/issues/420), indicating that the predefined size selections of Dexed's GUI are too small for high resolution displays such as 6K retina monitors.
In response to this issue, this commit introduces new size options: 200%, 300%, and 400%, extending the range beyond the current maximum of 150%.
This commit contains the file `ParamDialog.cpp` modified to include these new size options.

![4096x2160-300-percent](https://github.com/FulopNandor/dexed/assets/118678108/c5a50e0e-898e-4b64-b8f0-c6fe9fa57cef)
The picture above was created on a screen with a resolution of 4096x2160 when 300% size was set  in Dexed.

**Comment by ghost** [comment]
Thanks so much!
Is there a version I can test? I don't know anything about code. I want to test the 400% on my 6K screen.

**Comment by FulopNandor** [comment]
You’re very welcome!

As far as I know, there is currently no new official installable/executable version available that includes this new scaling option up to 400%. Perhaps Pascal Gauthier (User @asb2m10), the owner of the official Dexed repository, could provide an answer as to when the newer binary releases might be expected. Furthermore, if I understood correctly, you are using a Mac. This unfortunately exceeds my capabilities, as I only have an MS Windows 10 workstation and two Raspberry Pis. (Moreover, I’ve never had a Mac,so I have no experience with its use, let alone code development and compilation on it). 
Perhaps among your acquaintances or friends, there might be someone who would be willing to download the developer version containing the scaling up to 400% from my forked  repo (https://github.com/FulopNandor/dexed) and perform the build of the binaries you needed? I would also be curious about the result on a Mac with a 6K monitor!:-)

Best regards,

Nándor Fülöp

**Comment by ghost** [comment]
Is it not better to add this code the official and then the person there can compile it for everything?

**Comment by FulopNandor** [comment]
Yes, you are absolutely right - that is indeed my intention, too.

I started fixing a minor bug in Dexed that was identified recently, which I could solve in three steps. I have already completed two of these, when I read your request and thought that this could be resolved faster than the last of the aforementioned repair steps. So, I made this commit containing a 400% scaling option. But as soon as I manage to solve the mentioned last step, I will send a pull request to Paul Gauthier (@asb2m10), asking him to kindly accept my fixes - including the interjected commit containing the 400% scaling option - and merge them into the official branch. 
I hope that I can successfully complete the mentioned third step this week.

**Comment by ghost** [comment]
FUCK YOU MOTHERFUCKER

FUCKFACE
